ssl/test/bssl_shim.cc

```diff
@@ -306,7 +306,8 @@ static bool CheckListContains(const char *type,
   list_func(list.data(), list.size());
   for (const char *expected : list) {
     if (str == nullptr) {
-      break;
+      fprintf(stderr, "Unexpected %s: <null>\n", type);
```

Comment:
instead of adding an extra line and return statement.

Below use:
```
fprintf(stderr, "Unexpected %s: %s\n", type, str == nullptr ? "<null>" : str);
```